LumbApp/GUI/GUIController.cs

```diff
@@ -0,0 +1,124 @@
+﻿
+using KinectCoordinateMapping;
+using LumbApp.Enums;
+using LumbApp.Orquestador;
+using LumbApp.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows;
+using LumbApp.Expertos.ExpertoZE;
+
+namespace LumbApp.GUI
+{
+    public class GUIController
+    {
+        private Orquestador.Orquestador _orquestador { get; set; }
+        private MainWindow MainWindow { get; set; }
+        private SensorsCheck SensorsCheckPage { get; set; }
+        private IngresoDatosPracticante IngresoDatosPracticantePage { get; set; }
+
+        public GUIController(MainWindow mainWindow) {
+
+            MainWindow = mainWindow;
+        }
+
+        /// <summary>
+        /// Lo llama el main window para instanciar el orquestador
+        /// </summary>
+        public void Inicializar()
+        {
+            SensorsCheckPage = new SensorsCheck(this);
+            MainWindow.NavigationService.Navigate(SensorsCheckPage);
+            SensorsCheckPage.MostrarCheckeandoSensores();
+            _orquestador = new Orquestador.Orquestador(this);
+            _orquestador.Inicializar();
```

Comment:
Estoy viendo un warning en _orquestador.Inicializar() "Because this call is not awaited, execution of the current method continues before the call is completed."
Supongo que es lo que queremos, pero será que hay una mejor forma de hacerlo?

Reply:
Es lo mejor que encontramos hasta ahora, pero es cuestión de seguir probando

Reply:
Si, ignora ese warning, anda bien, justamente queremos que no espere que termine esa funcion, como dice carla fue la unica solucion que nos andubo por ahora